tools/build/android/webrtc-native/merge-archives.sh

```diff
@@ -131,6 +131,11 @@ while getopts l:m:o:vh OPTION; do
     esac
 done
 
+# Replace C:/ for /mnt/c/ if using gradle in windows
+MRWEBRTC_AAR=${MRWEBRTC_AAR/C://mnt/c}
+MRWEBRTC_AAR=${MRWEBRTC_AAR//\\//}
+echo $MRWEBRTC_AAR
```

Comment:
Can you please log a proper message rather than a value without context? It makes it difficult for users to understand what is this path they see. See `lib.sh` for many examples of output with `echo -e` and some ANSI escape for coloring if needed.